src/frontend/src/lib/components/receive/ReceiveActions.svelte

```diff
@@ -7,12 +7,13 @@
 	export let address: string;
 	export let qrCodeAriaLabel: string;
 	export let copyAriaLabel: string;
+	export let testCopyId: string | undefined = undefined;
```

Comment:
I noticed that this var is called in some files `testCopyId` and in others - `testIdCopy`. First of all, let's align the naming. Second of all, how about `copyButtonTestId`? It sounds a bit more cleaner.

Reply:
Fine for me 09097d39